docs/DeveloperGuide.md

```diff
@@ -509,15 +509,57 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   Use case ends
 
 * 1b. Patient does not exist
-    * 1b1. Omnihealth displays error message
+    * 1b1. OmniHealth displays error message
 
   Use case ends
 
 * 3a. No keywords matching can be found
-    * 3a1. Omnihealth displays error message
+    * 3a1. OmniHealth displays error message
 
   Use case ends
 
+**Use case: UC12 - Add patient appointment**
+
+**MSS**
+1. User requests to add an appointment with a specified date.
+2. An appointment is created for the patient.
+
+**Extensions**
+* 1a. Patient list is empty
```

Comment:
should this give show an error message to let the user know that the command didn't work as intended? 
Otherwise the user might assume the appointment was added successfully

Reply:
Yeah true will add.